packages/multichain-account-service/src/MultichainAccountWallet.ts

```diff
@@ -244,10 +244,27 @@ export class MultichainAccountWallet<
   }): Promise<void> {
     if (awaitAll) {
       const tasks = providers.map((provider) =>
-        provider.createAccounts({
-          entropySource: this.#entropySource,
-          groupIndex,
-        }),
+        provider
+          .createAccounts({
+            entropySource: this.#entropySource,
+            groupIndex,
+          })
+          .catch((error) => {
+            const sentryError = createSentryError(
+              `Unable to create account with provider "${provider.getName()}"`,
+              error,
+              {
+                groupIndex,
+                providerName: provider.getName(),
```

Comment:
I would just use `provider`, that's what we started to use for tracing as well, it's cleaner and more elegant IMO
```suggestion
                provider: provider.getName(),
```